sql-plugin/src/main/312db/scala/com/nvidia/spark/rapids/shims/DeltaLakeUtils.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021-2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids.shims
+
+import org.apache.spark.sql.execution.FileSourceScanExec
+
+object DeltaLakeUtils {
+  /* Check for _databricks_internal when running on Databricks */
+  def isDeltaLakeScan(f: FileSourceScanExec): Boolean = {
```

Comment:
There is an OSS version of DeltaLake. Should this be named something like isDatabricksDeltaLakeScan?

Reply:
I will update that. Thank you!